### ISSUE TITLE
Queue still prevents process from exiting

### ISSUE THREAD

**Issue by dustinlarimer** [keen/keen-tracking.js#84]
The internal event queue used by `client.defefEvent()` and `client.deferEvents()` still prevents processes from exiting in NodeJS, because the default `queueInterval` is greater than 0. This hanging behavior was presumed to be fixed in the previous release, but the issue is still present for use cases where the queue is not actually used.

As a temporary workaround, the following setting will allow processes to exit properly:

```javascript
const client = new Keen({ 
  /* Configure*/ 
});
client.queueInterval(0);
```

**Comment by dustinlarimer** [comment]
Cross-reference to this issue: https://github.com/keen/keen-js/issues/432

**Comment by dustinlarimer** [comment]
Fix for this released in [v1.2.1](https://github.com/keen/keen-tracking.js/releases/tag/v1.2.1) 👍
